### PR TITLE
Return latest evnen if version is already the latest

### PR DESCRIPTION
**Pull request by tan-tan-kanarek**

### CODE REVIEW

File: plugins/capture_space/lib/kCaptureSpaceVersionManager.php [hunk 1]
```diff
@@ -18,26 +18,23 @@ class kCaptureSpaceVersionManager{
...
 			$filename = __DIR__ . '/../config/' . self::INI_FILE_NAME;
 			self::$config = parse_ini_file($filename, true);
 		}
-		return self::$config;
-	}
-	
-	private static function getConfig($os, $osTypes, $version = null){
-		$config = self::initConfig();
+		
+		self::$config = self::initConfig();
```
Comment (gonenradai) [inline plugins/capture_space/lib/kCaptureSpaceVersionManager.php]: there's no such function initConfig on this class.

File: plugins/capture_space/lib/kCaptureSpaceVersionManager.php [hunk 2]
```diff
@@ -18,26 +18,23 @@ class kCaptureSpaceVersionManager{
...
 				return null;
 			}
-			$sections = array($version => $config[$version]);
+			$sections = array($version => self::$config[$version]);
 		}
 		else {
-			uksort($config, 'version_compare');
-			$sections = array_reverse($config, true);
+			uksort(self::$config, 'version_compare');
+			$sections = array_reverse(self::$config, true);
```
Comment (gonenradai) [inline plugins/capture_space/lib/kCaptureSpaceVersionManager.php]: This basically means INI must be written in ascending order of versions (lowest version - first in INI).
I'm OK with that, although it is error prone.
wouldn't be easier to maintain a [latest] section?

Comment (tan-tan-kanarek) [inline plugins/capture_space/lib/kCaptureSpaceVersionManager.php]: The order doesn't matter, I call uksort.

Comment (gonenradai) [inline plugins/capture_space/lib/kCaptureSpaceVersionManager.php]: right... missed that one.

**Pull request merged**